src/modules/auth/components/login/login-form/login-form.tsx

```diff
@@ -28,7 +33,16 @@ export function LoginForm() {
   }
 
   function onSubmit(data: LoginFormData) {
-    console.log(data);
+    axiosInstance
+      .post(ENDPOINTS.login, data)
+      .then((res) => res.data)
+      .then((res) => {
```

Comment:
Using two `then`s is unnecessary. Just extract data from response inside single `then`.

Reply:
fixed